docs/modules/ROOT/pages/how-tos/cloudscale/install.adoc

```diff
@@ -14,117 +16,157 @@ This how-to guide is still a work in progress and will change.
 It's currently very specific to VSHN and needs further changes to be more generic.
 --
 
+
 == Prerequisites
-* cloudscale.ch API token
-* `terraform`
-* `mc` https://docs.min.io/docs/minio-client-quickstart-guide.html[Minio client]
+
+* `docker`
+* `docker-compose`
+* `mc` https://docs.min.io/docs/minio-client-quickstart-guide.html[Minio client] (aliased to `mc` if necessary)
 * `jq`
 * `yq` https://mikefarah.gitbook.io/yq[yq YAML processor]
 * `openshift-install` https://cloud.redhat.com/openshift/install/metal/user-provisioned[OpenShift Installer]
-* Working local https://github.com/projectsyn/commodore[Commodore] setup
+* https://github.com/projectsyn/commodore[Commodore] docker-compose setup
```

Comment:
Instead of Docker Compose, could something like https://syn.tools/tuto/index.html#_using_the_commodore_docker_image be used?

Reply:
Yes and no: the setup of such a command would include quite some additional steps (duplicating the README) for API, Git URLs, credentials, etc.

Reply:
Maybe we should add a "local install" how-to to https://github.com/projectsyn/commodore/pull/147 ?

Reply:
I don't like the fact that you need to "install" commodore when you can just do a "docker run" with some parameters. As this documentation already is very VSHN specific, why not add the VSHN specific copy/paste command for the time being? We might also document how to use the `./commodore_command.sh` "officially" and say that the `.env` file is the Commodored "configuration file", which we could provide as an example in our VSHN wiki?

Reply:
Ok, I added the alias to use the docker image directly.